scapy/contrib/isotp.py

```diff
@@ -1813,12 +1814,14 @@ def scan_extended(sock, scan_range=range(0x7ff + 1), scan_block_size=100,
         # remove duplicate IDs
         id_list = list(set(id_list))
```

Comment:
I think `set` is redundant here.
`id_list = set(id_list)`

Reply:
In line 1847 as well
https://github.com/polybassa/scapy-1/pull/3/files#diff-a11ae4af4a4fabd503da745c5589f9beR1847